e2e/cypress/integration/api/iam/06_projects_api_compliance_update.ts

```diff
@@ -0,0 +1,236 @@
+import { describeIfIAMV2p1 } from '../../constants';
+import { uuidv4 } from '../../helpers';
+
+describeIfIAMV2p1('Compliance report project update tagging', () => {
+  const cypressPrefix = 'test-compliance-update';
+
+
+  const projectsWithRule = [
+    // This test is commented out because there is a current limit of 6 projects allowed
```

Comment:
should this still be commented out?

Reply:
looks like we haven't bumped the limit yet in the [bk script](https://github.com/chef/automate/blob/master/integration/tests/cypress_e2e.sh#L13)

Reply:
You are correct. I have not increased the limit yet.